client/src/lifecycleManager.js

```diff
@@ -225,13 +237,11 @@ class LifecycleManager extends LuigiClientBase {
   /**
    * Registers a listener called with the context object and the Luigi Core domain as soon as Luigi is instantiated. Defer your application bootstrap if you depend on authentication data coming from Luigi.
    * @param {Lifecycle~initListenerCallback} initFn the function that is called once Luigi is initialized, receives current context and origin as parameters
-   * @param {boolean} disableTpcCheck if set to `true` third party cookie check will be disabled via LuigiClient.
    * @memberof Lifecycle
    * @example
    * const initListenerId = LuigiClient.addInitListener((context) => storeContextToMF(context))
    */
-  addInitListener(initFn, disableTpcCheck) {
-    this.disableTpcCheck = disableTpcCheck;
+  addInitListener(initFn) {
```

Comment:
please keep the disableTcpCheck argument, it should still be possible to disable via js, e.g. in a test environment.
if it is true, just call document.head.setAttribute('disable-tpc-check') here instead of this.disableTpcCheck=true